react/react1/week1/my-app/src/App.js

```diff
@@ -0,0 +1,44 @@
+import './App.css';
+
+const tasks = [
+  { task: 'Get out of bed', date: 'Wed Sep 13 2017' },
+  { task: 'Brush teeth', date: 'Thu Sep 14 2017' },
+  { task: 'Eat breakfast', date: 'Fri Sep 15 2017' }
+];
+
+function Header() {
+  return <h1>ToDo List</h1>;
+}
+
+function Task(props) {
+  return (
+    <li class="task">
+      <input type="checkbox"></input>
+      <div>
+        <h3>{props.tasks.task}</h3>
+        <p>{props.tasks.date}</p>
+      </div>
+    </li>
+  );
+}
+
+function TaskList() {
+  return (
+    <ul>
+      {tasks.map((task) => (
+        <Task key={task.id} tasks={task} />
```

Comment:
You should add "key" property when creating lists in React.
Please read the docs to know why. https://github.com/KristinaGru/hyf-homework/pull/20